watchtower-plugin/src/retrier.rs

```diff
@@ -600,6 +601,19 @@ mod tests {
     const MAX_INTERVAL_TIME: u16 = 1;
     const MAX_RUN_TIME: f64 = 0.2;
 
+    #[macro_export]
```

Comment:
No need to export the macro.

Reply:
True, this was originally intended to go on `teos-common`, but I changed my mind 😅